src/rocket-pool/ObolRocketPoolStorage.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+import {Ownable} from "solady/auth/Ownable.sol";
+
+//https://docs.rocketpool.net/overview/contracts-integrations
+contract ObolRocketPoolStorage is Ownable {
```

Comment:
Is there a reason for this contract and why the below isn't gotten at runtime from the following the docs example? https://docs.rocketpool.net/developers/usage/contracts/contracts#interacting-with-rocket-pool

```
address public rocketPoolStorage;
  address public rocketPoolDeposit;
  address public rocketPoolMinipoolManager;
  address public rEth;
 ```

Reply:
We can use the RP storage now